hawkbit-dmf-amqp/src/test/java/org/eclipse/hawkbit/amqp/BaseAmqpServiceTest.java

```diff
@@ -49,57 +52,68 @@ public void setup() {
-
-        convertedActionUpdateStatus = baseAmqpService.convertMessage(null, ActionUpdateStatus.class);
-        assertThat(convertedActionUpdateStatus).as("Converted Object should be null when message is null").isNull();
+        assertThat(convertedActionUpdateStatus).isEqualToComparingFieldByField(actionUpdateStatus);
 
-        convertedActionUpdateStatus = baseAmqpService.convertMessage(new Message(null, new MessageProperties()),
-                ActionUpdateStatus.class);
-        assertThat(convertedActionUpdateStatus).as("Converted Object should be null when message body is null")
-                .isNull();
     }
 
     @Test
-    @Description("Verify that a conversion of a list from a message works")
-    public void convertMessageListTest() {
-        final List<ActionUpdateStatus> actionUpdateStatusList = new ArrayList<>();
-        for (int i = 0; i < 5; i++) {
-            final ActionUpdateStatus actionUpdateStatus = new ActionUpdateStatus();
-            actionUpdateStatus.setActionId(Long.valueOf(i));
-            actionUpdateStatus.setSoftwareModuleId(Long.valueOf(i));
-            actionUpdateStatusList.add(actionUpdateStatus);
+    @Description("Tests invalid null message content")
+    @ExpectEvents({ @Expect(type = TargetCreatedEvent.class, count = 0) })
+    public void convertMessageWithNullContent() {
+        try {
+            baseAmqpService.convertMessage(new Message(null, createJsonProperties()), ActionUpdateStatus.class);
+            fail("Expected MessageConversionException for inavlid JSON");
+        } catch (final MessageConversionException e) {
+            // expected
         }
 
-        final Message message = rabbitTemplate.getMessageConverter().toMessage(actionUpdateStatusList,
-                new MessageProperties());
-        List<ActionUpdateStatus> convertedActionUpdateStatus = baseAmqpService.convertMessageList(message,
-                ActionUpdateStatus.class);
+    }
+
+    @Test
+    @Description("Tests invalid empty message content")
+    @ExpectEvents({ @Expect(type = TargetCreatedEvent.class, count = 0) })
+    public void updateActionStatusWithEmptyContent() {
+        try {
+            baseAmqpService.convertMessage(new Message("".getBytes(), createJsonProperties()),
+                    ActionUpdateStatus.class);
+            fail("Expected MessageConversionException for inavlid JSON");
+        } catch (final MessageConversionException e) {
+            // expected
+        }
+    }
 
-        assertThat(convertedActionUpdateStatus).as("Converted Action Status list is wrong")
-                .hasSameClassAs(actionUpdateStatusList);
-        assertThat(convertedActionUpdateStatus).as("Converted Action Status list is wrong")
-                .hasSameSizeAs(actionUpdateStatusList);
+    private MessageProperties createJsonProperties() {
+        final MessageProperties messageProperties = new MessageProperties();
+        messageProperties.setContentType("json");
```

Comment:
Can't you use `messageProperties.setContentType(MessageProperties.CONTENT_TYPE_JSON)`?